Marlin/Marlin_main.cpp

```diff
@@ -2380,6 +2393,30 @@ void process_commands()
         lcd_lib_draw_shade(x, y, x + w, y + h);
         }
         break;
+    case 10008://M10008 - Display string at X,Y based on hex encoded chars (S), use I instead of S for inverted text, 
+               //         e.g. M10008 X10 Y10 S48656c6c6f2c20776f726c6421 displays "Hello, world!"
+        {
+          uint8_t x = 0, y = 0;
```

Comment:
This code should also clear out the '*' character, to prevent the hex2bin function trying to convert that.